package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Mon Mar 25 10:26:35 UTC 2019 - Guillaume GARDET <guillaume@opensuse.org>
+
+- Fix %arm links
```

Comment:
please add bugzilla or fate entry id in form of (boo#<num>, bsc#<num> or fate#<num>). and increase version. Thanks

Reply:
Done